portfolio/src/main/webapp/index.html

```diff
@@ -1,16 +1,42 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-    </div>
-  </body>
-</html>
+
+<head>
+	<meta charset="UTF-8">
```

Comment:
[indent 2 spaces at a time](https://google.github.io/styleguide/htmlcssguide.html#Indentation)

Reply:
Indentation fixed in commit 
609c261